pom.xml

```diff
@@ -93,4 +93,52 @@
     </dependency>
 
   </dependencies>
+
+  <build>
+    <plugins>
+
+      <!-- Override parent pom to suppress enforcing dependency convergence -->
```

Comment:
Dependency convergence errors are real problems with your library that could result in runtime errors depending how maven resolves them.  I wouldn't suppress them, instead I'd fix them.

Reply:
I should be able to fix these by excluding the sub-dependencies that conflict and then explicitly declaring them in the pom.